sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -853,9 +862,13 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     {
       protocolLayerOptions.FileContentDisposition = options.HttpHeaders.ContentDisposition;
     }
-    if (!options.HttpHeaders.ContentMd5.empty())
+    if (!options.HttpHeaders.ContentHash.Value.empty())
```

Comment:
How should we denote empty hash?

Now the algorithm is Md5, while the value is empty. This looks kind of weird.

Reply:
I can accept it since there is no way an empty value Md5 hash being valid.

Reply:
Should we add another enum `None` besides `Md5` and `Crc64`?

Reply:
To align with other enums, we should be naming it `Unknown`.